mokapot/mokapot.py

```diff
@@ -50,15 +57,22 @@ def main():
     logging.info("")
     logging.info("Starting Analysis")
     logging.info("=================")
+    logging.debug(f"Loaded plugins: {plugins.keys()}")
```

Comment:
**todo:** The logging module is odd and typically it is suggested to use %-style formatting:
```python
logging.debug("Loaded plugins: %s",  plugins.keys())
```